app/models/profile.rb

```diff
@@ -100,6 +100,11 @@ def strict_ial2_proofed?
     includes_phone_check?
   end
 
+  def has_proofed_before?
+    threshold = active? ? 2 : 1
+    Profile.where(user_id: user_id).where.not(activated_at: nil).count >= threshold
+  end
```

Comment:
@zachmargolis @mdiarra3 This code works for my purposes, but is this sound logic?

Proofing and "activated" appear synonymous but this is something I'd really appreciate confirmation on. It is possible that "active" and "proofed" are just coincidentally changed as similar times.

When we deactivate an account, we set `active: false` but leave `activated_at` alone. So if an account has `activated_at` set, we know they _have_ been activated (proofed?) previously.

That said, I think if I were to implement a `reproofed?` method, it would really be `has_proofed_before && active?`. I may want to change the usage above.